project_national/housing_characteristics/Misc Extra Refrigerator.tsv

```diff
@@ -1,9 +1,11 @@
-Dependency=Geometry Building Type RECS	Option=EF 10.2	Option=EF 10.5	Option=EF 15.9	Option=EF 17.6	Option=EF 6.7	Option=None	sample_count	sample_weight
-Mobile Home	0.002561	0.007348	0.031474	0.041683	0.002561	0.914373	533.000000	6842068.000000
-Multi-Family with 2 - 4 Units	0.001227	0.007565	0.019774	0.031681	0.001227	0.938526	921.000000	8975958.000000
-Multi-Family with 5+ Units	0.000121	0.001675	0.004964	0.020557	0.000121	0.972562	1900.000000	18886024.000000
-Single-Family Attached	0.005188	0.024873	0.046278	0.072301	0.005188	0.846172	887.000000	6698326.500000
-Single-Family Detached	0.014856	0.054054	0.086323	0.155556	0.014856	0.674355	7743.000000	71332376.000000
+Dependency=Geometry Building Type RECS	Option=EF 6.7	Option=EF 10.2	Option=EF 10.5	Option=EF 15.9	Option=EF 17.6	Option=EF 19.9	Option=None	sample_count	sample_weight
+Mobile Home	0.005345	0.000903	0.003979	0.026358	0.045026	0.004034	0.914355	533.000000	6842068.000000
+Multi-Family with 2 - 4 Units	0.002677	0.000391	0.003944	0.018584	0.032607	0.003038	0.938759	921.000000	8975958.000000
+Multi-Family with 5+ Units	0.000268	0.000045	0.000839	0.003599	0.020445	0.002291	0.972513	1900.000000	18886024.000000
+Single-Family Attached	0.011469	0.001771	0.013461	0.041609	0.078049	0.004957	0.848683	887.000000	6698326.500000
+Single-Family Detached	0.033185	0.005329	0.028505	0.082042	0.157780	0.017258	0.675901	7743.000000	71332376.000000
 # Created by: sources\recs\2009\tsv_maker.py
 # Source: Constructed using U.S. EIA 2009 Residential Energy Consumption Survey (RECS) microdata.
-# Source: Age of refrigerator converted to efficiency levels using ENERGYSTAR shipment-weighted efficienciesby year.
\ No newline at end of file
+# Source: Age of refrigerator converted to efficiency levels using ENERGYSTAR shipment-weighted efficienciesby year data from Home Energy Score: http://hes-documentation.lbl.gov/. Check the comments in: HES-Refrigerator_Age_vs_Efficiency.tsv
```

Comment:
White space `efficienciesby` --> `efficiencies by`. Needs to be fixed in resstock-estimation